libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5173,6 +5173,32 @@ MAV_RESULT GCS_MAVLINK::handle_do_set_safety_switch_state(const mavlink_command_
     }
 }
 
+#if AP_MAVLINK_MAV_CMD_AWR_SYS_CMP_ID_ENABLED
```

Comment:
Uh - that might have been a typo on my part.  No sure what `AWR` is here :-)  Possible `DO`?!